integration/cpp/test/test_collection_sql.cpp

```diff
@@ -110,20 +116,23 @@ TEST_CASE("integration::cpp::test_collection::sql::base") {
             auto cur = dispatcher->execute_sql(session,
                                                "SELECT * FROM TestDatabase.TestCollection "
                                                "WHERE count > 90;");
+            REQUIRE(cur->is_success());
             REQUIRE(cur->size() == 9);
         }
         {
             auto session = otterbrix::session_id_t();
             auto cur = dispatcher->execute_sql(session,
                                                "DELETE FROM TestDatabase.TestCollection "
                                                "WHERE count > 90;");
+            REQUIRE(cur->is_success());
             REQUIRE(cur->size() == 9);
         }
         {
             auto session = otterbrix::session_id_t();
             auto cur = dispatcher->execute_sql(session,
                                                "SELECT * FROM TestDatabase.TestCollection "
                                                "WHERE count > 90;");
+            REQUIRE(!cur->is_success());
```

Comment:
https://github.com/catchorg/Catch2/blob/devel/docs/assertions.md
```suggestion
            REQUIRE_FALSE(cur->is_success());
```